mswindows/GRASS-Installer.nsi.tmpl

```diff
@@ -896,22 +839,8 @@ Section "Important Microsoft Runtime DLLs" SecMSRuntime
 	;Set the size (in KB) of the unpacked archive file
 	AddSize 13500
 	
-	StrCpy $HTTP_PATH "http://download.osgeo.org/osgeo4w/${PLATFORM}/release/msvcrt/"
-        !if ${PLATFORM} == "x86_64"
-            StrCpy $ARCHIVE_NAME "msvcrt-1.0.1-9.tar.bz2"
-            StrCpy $ARCHIVE_NAME2008 "msvcrt2008-1.0.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2010 "msvcrt2010-1.0.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2012 "msvcrt2012-1.0.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2013 "msvcrt2013-1.0.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2015 "msvcrt2015-1.0-1.tar.bz2"
-        !else
-            StrCpy $ARCHIVE_NAME "msvcrt-1.0.1-14.tar.bz2"
-            ;StrCpy $ARCHIVE_NAME2005 "msvcrt2005-1.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2008 "msvcrt2008-1.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2010 "msvcrt2010-1.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2013 "msvcrt2013-1.0-1.tar.bz2"
-            StrCpy $ARCHIVE_NAME2015 "msvcrt2015-1.0-1.tar.bz2"
-        !endif
+	StrCpy $HTTP_PATH "http://download.osgeo.org/osgeo4w/v2/${PLATFORM}/release/msvcrt2019/"
```

Comment:
where do we define **${PLATFORM}**?

Reply:
@hellik https://github.com/OSGeo/grass/blob/releasebranch_7_8/mswindows/GRASS-Installer.nsi.tmpl#L45

I tried to keep modifications in releasebranch_7_8 minimalistic. I will prepare separated PR for main branch.